test/test_utils.py

```diff
@@ -12,11 +12,13 @@
 import torch.nn as nn
 import torch.utils.data
 import torch.cuda
+import torchvision.models as models
```

Comment:
Don’t we have to install `torchvision` for the tests to be executed successfully? I think the CI docker images don’t do this.